File: modul_praktikum_python/contoh_code/pascal/menu_geometri_case.pas
uses crt; 
const 
   pi = 3.14; 
var 
  pilih : integer; 
  sisi,jari,tinggi : real; 
begin 
  writeln(‘         <<< Menu >>>        ’); 
  writeln; 
  writeln(‘1. Menghitung Isi Kubus’); 
  writeln(‘2. Menghitung Luas Lingkaran’); 
  writeln(‘3. Menghitung Isi Silisnde’); 
  writeln; 
  write(‘Pilih Nomor : ’);readln(pilih); 
  case pilih of 
  1 : begin 
       write(‘Panjang Sisi Kubus : ’);readln(sisi); 
       writeln(‘Isi Kubus : ’,sisi*sisi*sisi); 
      end; 
  2 : begin 
       write(‘Jari-jari lingkaran : ’);readln(jari); 
       writeln(‘Luas Lingkaran : ’,pi*jari*jari); 
      end; 
  3 : begin 
       write(‘Jari-jari lingkaran : ’);readln(jari); 
       write(‘Tinggi Silinder : ’);readln(tinggi); 
       writeln(‘Isi Silinder : ’,pi*jari*jari*tinggi); 
      end; 
  end;  
end.

File: modul_praktikum_python/contoh_code/python/menu_geometri_case.py
"""Menu hitung kubus, luas lingkaran, isi silinder (satu kali pilihan)."""

PI = 3.14


def main() -> None:
    print("         <<< Menu >>>        ")
    print()
    print("1. Menghitung Isi Kubus")
    print("2. Menghitung Luas Lingkaran")
    print("3. Menghitung Isi Silisnde")
    print()
    pilih = int(input("Pilih Nomor : "))
    if pilih == 1:
        sisi = float(input("Panjang Sisi Kubus : "))
        print("Isi Kubus :", sisi * sisi * sisi)
    elif pilih == 2:
        jari = float(input("Jari-jari lingkaran : "))
        print("Luas Lingkaran :", PI * jari * jari)
    elif pilih == 3:
        jari = float(input("Jari-jari lingkaran : "))
        tinggi = float(input("Tinggi Silinder : "))
        print("Isi Silinder :", PI * jari * jari * tinggi)


if __name__ == "__main__":
    main()
